transports/quartz/src/main/java/org/mule/transport/quartz/QuartzMessageReceiver.java

```diff
@@ -87,7 +87,7 @@ protected void doStart() throws MuleException
                 throw new IllegalArgumentException(CoreMessages.objectIsNull(QuartzConnector.PROPERTY_JOB_CONFIG).getMessage());
             }
 
-            JobDetail jobDetail = new JobDetail();
+            JobDetailImpl jobDetail = new JobDetailImpl();
```

Comment:
again the same as in the dispatcher. COnsider having factory methods to create these things